packages/core/strapi/lib/services/entity-validator/validators.js

```diff
@@ -166,7 +166,9 @@ const stringValidator = composeValidators(
   addUniqueValidator
 );
 
-const emailValidator = composeValidators(stringValidator, validator => validator.email());
+const emailValidator = composeValidators(stringValidator, validator =>
```

Comment:
Am I missing something or does this not change anything but move something to a new line?

Reply:
Hi @derrickmehaffy ,
fix `validator.email()` to `validator.email().required()` to line 170.

Reply:
I am blind, thanks :laughing: